manager/integration/tests/test_recurring_job.py

```diff
@@ -421,7 +421,7 @@ def test_recurring_job_in_storageclass(set_random_backupstore, client, core_api,
     for pod_name in pod_names:
         write_pod_volume_random_data(core_api, pod_name, volume_data_path, 2)
 
-    time.sleep(150)  # 2.5 minutes
+    time.sleep(140)  # 2.5 minutes - 10 secends
```

Comment:
If you already made the change in create_recurring_jobs, why need to change the sleep time? (including the all changes in this module)